internal/buildscripts/packaging/msi/msi-builder/Dockerfile

```diff
@@ -1,10 +1,10 @@
-# The quay.io/signalfx/wix-dev:latest base image is tagged and pushed from felfert/wix:latest
-# in order to pin the image in case the original image's latest tag is updated.
+# The quay.io/signalfx/wix-dev:fleetdm base image is tagged and pushed from fleetdm/wix:latest
+# in order to pin the image in case the original image's latest tag is updated or removed.
+# https://github.com/fleetdm/fleet/blob/main/LICENSE (MIT Expat)
 
-FROM quay.io/signalfx/wix-dev:latest
+FROM quay.io/signalfx/wix-dev:fleetdm
```

Comment:
may be worth holding off on this until this tag is from our own image build?

Reply:
Similar to `quay.io/signalfx/wix-dev:latest`, `quay.io/signalfx/wix-dev:fleetdm` was just tagged and pushed from the original image without any changes.